bundle/regal/rules/bugs/constant_condition.rego

```diff
@@ -17,16 +17,9 @@ _operators := {"equal", "gt", "gte", "lt", "lte", "neq"}
 
 _rules_with_bodies := [rule |
 	some rule in input.rules
-	not probably_no_body(rule)
+	not ast.no_body(rule)
```

Comment:
wonder if there's a way to write this where the function is `has_body` and then not has_body might be easier to understand at a glance.

Reply:
Yeah, that's better. I'll merge this but can fix in the next round of tweaks :)